lib/ansible/modules/cloud/openstack/os_port.py

```diff
@@ -217,6 +219,15 @@
 from ansible.module_utils.openstack import openstack_full_argument_spec, openstack_module_kwargs, openstack_cloud_from_module
 
 
+try:
+    from collections import OrderedDict
+except ImportError:
+    try:
+        from ordereddict import OrderedDict
+    except ImportError:
+        pass
```

Comment:
This needs to be a failure, the `OrderedDict` object is used.

Maybe just do a straight import with the try / except

``` python
try:
    from collections import OrderedDict
except ImportError:
    from ordereddict import OrderedDict
```

Reply:
Ah, Sloppy me didn't fix my initial try at the issue! Thanks!

Reply:
Or, this might've been the reason:
` ansible-test / sanity / import-python-2.6
failure: The test `ansible-test sanity --test import --python 2.6` failed with 1 error:
lib/ansible/modules/cloud/openstack/os_port.py:224:0: ImportError: No module named ordereddict`

oredereddict needs to be installed for python2.6